### DISCUSSION TITLE
Can you gide me how to use this

### DISCUSSION THREAD

**Discussion by Adil091** [lowlighter/metrics#718] [Q&A] [answered]
i am a newbe coder and i am using github first time.i dont know how to use this templets to my readme profile please guide me step by step to use the templte for my profile

**Comment by lowlighter** [comment, marked as answer]
Hello 👋 !

You can read the full [documentation setup](https://github.com/lowlighter/metrics/blob/master/.github/readme/partials/setup/action/setup.md) here, or on the [`README.md`](https://github.com/lowlighter/metrics#-how-to-use) by clicking on `💬 How to setup? (click to expand)` to see instructions
